vignettes/response-FDA IR-pilot3.qmd

```diff
@@ -0,0 +1,243 @@
+format: 
+  pdf:
+    toc: false
+    number-sections: true
+    include-before:
+      - '\newpage{}'
+editor: source
+---
+
+{{< pagebreak >}}
+
+![](images/RConsortium_Horizontal_Pantone.png){width="228"}
+
+::: {.flushright data-latex=""}
+April 12, 2024
+:::
+
+Food and Drug Administration
+Center for Drug Evaluation and Research 5901-B Ammendale Road
+Beltsville, MD 20705-1266
+
+
+Re: Response to FDA’s Statistical Review and Evaluation Summary of the R Consortium R submission Pilot 3 eCTD submission
+
+
+Dear Sir/Madam:
+
+
+On February 2, 2024, the R Consortium R submission Working Group received FDA’s Statistical Review and Evaluation Summary of the R Consortium R submission Pilot 3 submission. The working group would like to thank FDA staff for the comprehensive review and thoughtful recommendations. We are thrilled to learn that the FDA staff was able to reproduce the ADaM (Analysis Data Model) datasets as well as the analysis outputs using the R programs and the proprietary R package submitted through the eCTD portal.
+
+The purpose of this letter is to provide the working group’s responses to FDA’s Statistical Review and Evaluation Summary of the R Consortium R submission Pilot 3. The following information is provided in response to the findings from FDA Staff’s review :
+
+{{< pagebreak >}}
+
+[**Agency Comment 1**]{.underline}
+
+The agency found issues switching between R versions. The FDA would like the wg \[working group\] to explore the impact of different versions of R, RStudio and RTools, as well as Linux vs. Windows
+
+[**The R Consortium R Submission Working Group Response**]{.underline}
+
+It is recommended by the Working group to download and install the R and R Studio versions specified in the ADRG. Switching to other versions, not specified, could most likely be the cause of issues when running the R ADaM and analysis output programs. Certain versions of the R packages used in these analysis programs only correspond to the specified R version.
+
+Regarding RTools, referencing the installation instructions on its website,
+
+*"Rtools42 is only needed for installation of R packages from source or building R from source. R can be installed from the R binary installer and by default will install binary versions of CRAN packages, which does not require Rtools42.*
+
+*Moreover, online build services are available to check and build R packages for Windows, for which again one does not need to install Rtools42 locally. The Winbuilder check service uses identical setup as the CRAN incomming packages checks and has already all CRAN and Bioconductor packages pre-installed."*^1^
+
+As discussed in the R Consortium R Submission working group, *"It appears that the most practical path forward as to what preemptive measures sponsors can take to match submission environments with the FDA test environment is to work with the FDA to lock down the environment as best as can be done just prior to submission."*^2^
```

Comment:
```suggestion
As discussed in the R Consortium R Submission Working Group, *"It appears that the most practical path forward as to what preemptive measures sponsors can take to match submission environments with the FDA test environment is to work with the FDA to lock down the environment as best as can be done just prior to submission."*^2^
```

Reply:
Also - shoudl we just remove "It appears that" at the beginning of the sentence and just say "The most practical..."

Reply:
Fine to leave as I'm quoting the meeting minutes here . 
![image](https://github.com/RConsortium/submissions-pilot3-adam/assets/95468392/92c2fcf9-6557-4c4d-a3a4-b53d6566de89)